src/index.ts

```diff
@@ -11,6 +11,7 @@ import { Base58 } from '@ethersproject/basex'
 import { toUtf8Bytes } from '@ethersproject/strings'
 import { EthrDidController, interpretIdentifier, REGISTRY } from 'ethr-did-resolver'
 import { Resolvable } from 'did-resolver'
+import { address, MetaSignature } from 'ethr-did-resolver/lib/helpers'
```

Comment:
I have also noticed that we have not exported these in `ethr-did-resolver` and that it is breaking your pipeline. I have not much experience with writing packages so what do we do about it? I guess we can either redeclare them here or export them in the `index.ts` (?) in `ethr-did-resolver`. Wdyt?

Reply:
It's ok to redeclare until they get exported from there.. they are simple enough types, and unlikely to change and cause a conflict